profiles/default/iceberg/snowflake-views.sql.ftl

```diff
@@ -0,0 +1,6 @@
+<#if iceberg?? && iceberg["engines"]["snowflake"]??>
+<#list iceberg["engines"]["snowflake"]["views"] as statement>
+${statement["sql"]};
```

Comment:
I'll move this to the snowflake engine, it makes more sense for it to be there. This should have schema for iceberg related settings, like registering schemas in aws glue et al. I'll make the change.

Reply:
Agreed, each query engine should be it's own folder so we can put the DDL (if any) and view definitions there.